File: app/src/main/java/com/example/olgadominguez/todoachieved/di/ApplicationModule.java
package com.example.olgadominguez.todoachieved.di;

import android.content.Context;

import com.example.olgadominguez.todoachieved.TodoApplication;

import javax.inject.Singleton;

import dagger.Module;
import dagger.Provides;

@Module
public class ApplicationModule {

    @Provides
    @Singleton
    Context provideContext(TodoApplication application) {
        return application;
    }
}

File: app/src/main/java/com/cottondroid/todoachieved/di/ApplicationModule.kt
package com.cottondroid.todoachieved.di

import android.content.Context
import com.cottondroid.todoachieved.TodoApplication
import dagger.Module
import dagger.Provides
import javax.inject.Singleton

@Module
class ApplicationModule {
    @Provides
    @Singleton
    fun provideContext(application: TodoApplication): Context {
        return application
    }
}